### PR TITLE
Shorten and improve README

### PR DESCRIPTION
**Pull request by jvican**
Link to our online documentation and only give a brief explanation of what
bloop is in the README. Shorter is better. In the future we can add more
specific developer docs in the README, but for now it's not necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,68 +6,33 @@
...
-$ cd ..
-# paste the version number and SHA obtained above in the following command:
-$ bin/install.py --dest $HOME/.bloop --nailgun <nailgun-commit-sha> --version <version>
-```
+Bloop is a Scala command-line tool and build server developed at the Scala
+Center to make the compile and test feedback loop as tight as possible, with
+a focus on developer productivity.
 
-## Tab-completion
+### Tigth feedback loop
```
Comment (Duhemm) [inline README.md]: Typo: Tight

File: README.md [hunk 2]
```diff
@@ -6,68 +6,33 @@
...
-```sh
-$ mkdir -p ~/.zsh/completion
-$ cp etc/_bloop ~/.zsh/completion/
-$ echo 'fpath=(~/.zsh/completion $fpath)' >> ~/.zshrc
-$ echo 'autoload -Uz compinit ; compinit' >> ~/.zshrc
-```
+### Documentation
 
-Closing and reopening your shell should make bloop tab-completions available.
+Our documentation lives in the website.
```
Comment (Duhemm) [inline README.md]: Link to the website?

File: README.md [hunk 3]
```diff
@@ -6,68 +6,33 @@
...
 
-[installation-script]: https://raw.githubusercontent.com/scalacenter/bloop/master/bin/install.sh
-[zsh-completions]: https://raw.githubusercontent.com/scalacenter/bloop/master/etc/_bloop
-[bloop-release-post]: http://www.scala-lang.org/blog/2017/11/30/bloop-release.html
+1. [Read users documentation](https://scalacenter.github.io/bloop/docs/) to
```
Comment (Duhemm) [inline README.md]: To **read** about the project

File: website/content/docs/how-to-use-bloop.md [hunk 4]
```diff
@@ -19,6 +19,29 @@ client is a python script that implements Nailgun's protocol and communicates
...
+$ mkdir -p ~/.zsh/completion
+$ cp etc/_bloop ~/.zsh/completion/
+$ echo 'fpath=(~/.zsh/completion $fpath)' >> ~/.zshrc
+$ echo 'autoload -Uz compinit ; compinit' >> ~/.zshrc
+```
+
+Now, reload your shell (or close and open it again) to make zsh pick up bloop
+completions.
+
+[zsh-completions]: https://raw.githubusercontent.com/scalacenter/bloop/master/etc/_bloop
```
Comment (Duhemm) [inline website/content/docs/how-to-use-bloop.md]: Maybe add a link to that section in the installation instructions? Or move that directly to the installation instructions?

Comment (jvican) [reply website/content/docs/how-to-use-bloop.md]: I've thought about it. Do you think it's worth it to add it now or to do it when we have thought everything through?

**Pull request merged**